This is the failure:

```
22:07:59 FAILED: bin/sourcekitd-test 
22:07:59 : && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang++  -Wno-unknown-warning-option -Werror=unguarded-availability-new -fno-stack-protector -fPIC -fvisibility-inlines-hidden -Werror=date-time -Werror=unguarded-availability-new -std=c++14 -Wall -Wextra -Wno-unused-parameter -Wwrite-strings -Wcast-qual -Wmissing-field-initializers -Wimplicit-fallthrough -Wcovered-switch-default -Wno-class-memaccess -Wno-noexcept-type -Wnon-virtual-dtor -Wdelete-non-virtual-dtor -Wstring-conversion -fdiagnostics-color -Werror=switch -Wdocumentation -Wimplicit-fallthrough -Wunreachable-code -Woverloaded-virtual -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -O3  -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -Wl,-search_paths_first -Wl,-headerpad_max_install_names -mmacosx-version-min=10.9    -target x86_64-apple-macosx10.9 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -arch x86_64 -F /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/../../../Developer/Library/Frameworks -mmacosx-version-min=10.9 -Wl,-dead_strip -Xlinker -exported_symbol -Xlinker _main tools/SourceKit/tools/sourcekitd-test/CMakeFiles/sourcekitd-test.dir/sourcekitd-test.cpp.o tools/SourceKit/tools/sourcekitd-test/CMakeFiles/sourcekitd-test.dir/TestOptions.cpp.o  -o bin/sourcekitd-test -L/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/./lib -Wl,-rpath,/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/./lib -Wl,-rpath,/Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/swift-macosx-x86_64/lib lib/libSourceKitSupport.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libclangRewrite.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libclangLex.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libclangBasic.a lib/sourcekitd.framework/Versions/A/sourcekitd /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMCore.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMRemarks.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMBitstreamReader.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMMC.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMBinaryFormat.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMDebugInfoCodeView.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMDebugInfoMSF.a lib/libswiftBasic.a /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMSupport.a -lz -lcurses -lm /Users/buildnode/jenkins/workspace/swift-PR-osx/branch-master/buildbot_incremental/llvm-macosx-x86_64/lib/libLLVMDemangle.a lib/libswiftDemangling.a lib/libswiftSyntax.a && :
22:07:59 Undefined symbols for architecture x86_64:
22:07:59   "vtable for llvm::opt::InputArgList", referenced from:
22:07:59       llvm::opt::InputArgList::~InputArgList() in TestOptions.cpp.o
22:07:59   NOTE: a missing vtable usually means the first non-inline virtual member function has no definition.
22:07:59   "llvm::opt::OptTable::OptTable(llvm::ArrayRef<llvm::opt::OptTable::Info>, bool)", referenced from:
22:07:59       sourcekitd_test::TestOptions::parseArgs(llvm::ArrayRef<char const*>) in TestOptions.cpp.o
22:07:59   "llvm::opt::InputArgList::releaseMemory()", referenced from:
22:07:59       llvm::opt::InputArgList::~InputArgList() in TestOptions.cpp.o
22:07:59   "llvm::opt::OptTable::PrintHelp(llvm::raw_ostream&, char const*, char const*, bool, bool) const", referenced from:
22:07:59       sourcekitd_test::TestOptions::parseArgs(llvm::ArrayRef<char const*>) in TestOptions.cpp.o
22:07:59   "llvm::opt::OptTable::ParseArgs(llvm::ArrayRef<char const*>, unsigned int&, unsigned int&, unsigned int, unsigned int) const", referenced from:
22:07:59       sourcekitd_test::TestOptions::parseArgs(llvm::ArrayRef<char const*>) in TestOptions.cpp.o
22:07:59   "llvm::opt::OptTable::~OptTable()", referenced from:
22:07:59       sourcekitd_test::TestOptions::parseArgs(llvm::ArrayRef<char const*>) in TestOptions.cpp.o
22:07:59   "llvm::opt::Arg::getAsString(llvm::opt::ArgList const&) const", referenced from:
22:07:59       sourcekitd_test::TestOptions::parseArgs(llvm::ArrayRef<char const*>) in TestOptions.cpp.o
22:07:59 ld: symbol(s) not found for architecture x86_64
```

(from https://ci.swift.org/job/swift-PR-osx/18140/console)

Seems that some of those libraries are necessary in macOS.

--- a/tools/SourceKit/tools/sourcekitd-test/CMakeLists.txt
+++ b/tools/SourceKit/tools/sourcekitd-test/CMakeLists.txt
@@ -5,7 +5,7 @@ swift_add_public_tablegen_target(sourcekitdTestOptionsTableGen)
 add_sourcekit_executable(sourcekitd-test
   sourcekitd-test.cpp
   TestOptions.cpp
-  LLVM_LINK_COMPONENTS core option coverage lto
+  LLVM_LINK_COMPONENTS option coverage lto
 )
 target_link_libraries(sourcekitd-test PRIVATE
   SourceKitSupport

